.github/dependabot.yml

```
@@ -0,0 +1,63 @@
+    schedule:
+      interval: "weekly"
+      day: "wednesday"
+      time: "09:00"
+    open-pull-requests-limit: 3
+    assignees:
+      - "harperreed"
+    commit-message:
+      prefix: "ci"
+    labels:
+      - "dependencies"
+      - "github-actions"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)